Material.Styles/ColorZone.xaml

```diff
@@ -2,131 +2,47 @@
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=PrimaryMid] Button,
-                     cc|ColorZone[Mode=PrimaryMid] ToggleButton">
-        <Setter Property="Foreground" Value="{DynamicResource PrimaryHueMidForegroundBrush}" />
-        <Setter Property="assists:ToggleButtonAssist.UncheckedForeground" Value="{DynamicResource PrimaryHueMidForegroundBrush}"/>
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=PrimaryDark]">
-        <Setter Property="Background" Value="{DynamicResource PrimaryHueDarkBrush}" />
-        <Setter Property="Foreground" Value="{DynamicResource PrimaryHueDarkForegroundBrush}" />
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=PrimaryDark] Button,
-                     cc|ColorZone[Mode=PrimaryDark] ToggleButton">
-        <Setter Property="Foreground" Value="{DynamicResource PrimaryHueDarkForegroundBrush}" />
-        <Setter Property="assists:ToggleButtonAssist.UncheckedForeground" Value="{DynamicResource PrimaryHueDarkForegroundBrush}"/>
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=Accent]">
-        <Setter Property="Background" Value="{DynamicResource SecondaryHueMidBrush}" />
-        <Setter Property="Foreground" Value="{DynamicResource SecondaryHueMidForegroundBrush}" />
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=Accent] Button,
-                     cc|ColorZone[Mode=Accent] ToggleButton">
-        <Setter Property="Foreground" Value="{DynamicResource SecondaryHueMidForegroundBrush}" />
-        <Setter Property="assists:ToggleButtonAssist.UncheckedForeground" Value="{DynamicResource SecondaryHueMidForegroundBrush}"/>
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=Light]">
-        <Setter Property="Background" Value="{DynamicResource MaterialDesignLightBackground}" />
-        <Setter Property="Foreground" Value="{DynamicResource MaterialDesignLightForeground}" />
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=Light] Button,
-                     cc|ColorZone[Mode=Light] ToggleButton">
-        <Setter Property="Foreground" Value="{DynamicResource MaterialDesignLightForeground}" />
-        <Setter Property="assists:ToggleButtonAssist.UncheckedForeground" Value="{DynamicResource MaterialDesignLightForeground}"/>
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=Dark]">
-        <Setter Property="Background" Value="{DynamicResource MaterialDesignDarkBackground}" />
-        <Setter Property="Foreground" Value="{DynamicResource MaterialDesignDarkForeground}" />
-    </Style>
-    
-    <Style Selector="cc|ColorZone[Mode=Dark] Button,
-                     cc|ColorZone[Mode=Dark] ToggleButton">
-        <Setter Property="Foreground" Value="{DynamicResource MaterialDesignDarkForeground}" />
-        <Setter Property="assists:ToggleButtonAssist.UncheckedForeground" Value="{DynamicResource MaterialDesignDarkForeground}"/>
-    </Style>
-
+<!-- Compatibility issue: require preloaded MaterialTheme
```

Comment:
I was trying some new technique to provide background and foreground brushes, because the style way is not works sometimes. I could rollback those changes if you dont like the newer ColorZone

Reply:
I need to see more details, I'll look at it later.

Reply:
please let me know this techniqhe is helpful or not thx